examples/export.html

```diff
@@ -82,43 +71,46 @@
         ];
 
         // customize the function that starts up the export
-        function fetchStatusUrl(opts) {
-          $.ajax({
-            url: opts.exportStartUrl,
-            crossDomain: true,
-            type: 'POST',
-            data: {
-              collection: opts.collection,
-              scale: prompt("Choose a scale or use the default (cm per pixel):", 100) || opts.scale,
-              upload: true,
-            },
-            success: function(data) { opts.handleStatusRes(data, opts) }, // this handles the initial response
-          });
-        }
+        const fetchStatusUrl = (mergedOpts) => {
```

Comment:
These look good, but without the upstream server working, I guess I would not have merged these changes without testing it against the cloud export server!

Reply:
Can you elaborate please? I have some catching up to do! Is there anything that i should focus on immediately?